eeg_cofitage/README

```diff
@@ -0,0 +1,71 @@
+﻿Overview
+--------
+
+This is the “COFITAGE” dataset. The aim is to collect sleep EEG data from 100 to 120 healthy older 
+participants (aged 50-70), in order to quantify sleep disruption, evaluate its impact on cognition
+and assess its relationship with brain amyloid and tau proteins deposition in preclinical AD.
+
+We would like to determine which sleep-wake regulation parameters are associated with biomarkers 
+and cognitive status at baseline and at a 24 months follow-up of cognitive performances.
+
+
+Citing this dataset
+-------------------
+
+This dataset and related analyses are unpublished. Please do not use/cite given data other than
+BIDS dataset example.
+
+
+Format
+------
+
+It is formatted according to BIDS version 1.2.0 and the BIDS standard extension
+for EEG (BEP006). The full description of the standard and extension which can be found here:
+https://bids-specification.readthedocs.io/en/latest/
+
+This dataset is originally created in Embla 3.4 format. It is converted to EDF+ and applied to BIDS standard by a private tool. The configuration and log files are stored in the sourcedata folders.
```

Comment:
`by a private tool` --> with "private" do you mean that you don't want to disclose the conversion code?

I think it would be great to host this tool in some accessible place (like your personal GitHub repository, if you are the developer). Then we could refer to the tool and it'd be more transparent how the conversion took place.

Reply:
Actually the tool is private in sense is available only within the laboratory. I will make it public, but I need first obtain a license from university.

Reply:
dounds good, looking forward to it